docs/_running-tests/safari.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: page
+title: Safari
+---
+To run Safari on macOS, some manual setup is required:
+
+  * Disable "Block pop-up windows" in the security preferences.
+
+  * Ensure that "Allow Remote Automation" is enabled in the "Develop" menu.
+
+  * Import the web-platform.test certificate and root certificate authority in
+    `tools/certs/` into the keychain of the user that will run the tests, and
+    manually mark the root as trusted.
+
+  * Set `OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES` in your environment. This is a
```

Comment:
Is this specific to Safari support, or is this a larger wpt issue? safaridriver itself doesn't fork.

Reply:
```diff
+  * Set `OBJC_DISABLE_INITIALIZE_FORK_SAFETY=YES` in your environment. This is a
```
> Is this specific to Safari support, or is this a larger wpt issue?

It’s a larger wpt issue, not specific to Safari support. It seems to be an issue/bug in Python2 around how it handles multiprocessing on MacOS. See https://github.com/w3c/web-platform-tests/issues/9007 and https://bugs.python.org/issue30837

Reply:
Larger `wptrunner` issue (because it does fork).

Reply:
Yeah, if we had docs for Chrome on macOS it would have to say the same thing. A fix for that issue would be good, but until then documenting it helps.s